src/components/time/__tests__/__snapshots__/ReadableTime.test.js.snap

```diff
@@ -1,10 +1,10 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`components/time/ReadableTime should use default relative threshold if not provided 1`] = `"in 1,800,000 seconds"`;
```

Comment:
this should probably not be changing. double check your node version; for buie it should use >=12.x 

cc @alexkrolick

Reply:
Resolved.